### PR TITLE
DOC: Link rfft

### PR DESCRIPTION
**Pull request by larsoner**
Helps link things a bit for #2487. Just need to ensure that CircleCI links properly.

### CODE REVIEW

**Comment by jakirkham** [comment]
Should there be a link for the `irfft` function as well?

**Comment by larsoner** [comment]
I didn't think it would be too helpful since usually people go rfft->irrft (using NumPy or SciPy), but thinking about it more, people could start with freq-domain data and need to get to time, so yeah I'll add it.

File: scipy/fftpack/basic.py [hunk 1]
```diff
@@ -395,7 +395,7 @@ def rfft(x, n=None, axis=-1, overwrite_x=False):
 
     See Also
     --------
-    fft, irfft, scipy.fftpack.basic
```
Comment (pv) [inline scipy/fftpack/basic.py]: Maybe remove scipy.fftpack.basic, not supposed to be a public module

**Comment by larsoner** [comment]
@pv @jakirkham comments addressed

**Pull request merged**